pkg/controller/jenkins/reconciler.go

```diff
@@ -44,6 +44,7 @@ const (
 	JenkinsJnlpServiceSuffix = "-jnlp"
 	JenkinsImage             = "image-registry.openshift-image-registry.svc:5000/openshift/jenkins"
 	JenkinsContainerName     = "jenkins"
+	JenkinsContainerMemory   = "4Gi"
```

Comment:
4Gi is overkill no ? Let's keep it 1Gi for default ? We should have an option in the Jenkins CR for this.

Reply:
yeah, 1Gi should be good I feel.

Reply:
with 1Gi, Jenkins is very slow to start.
I will let it to 1Gi per your recommendations. And, we will find time to expose it through the CR.